doc/man1/openssl-cmp.pod.in

```diff
@@ -1029,11 +1030,12 @@ The certificate verification options
 B<-verify_hostname>, B<-verify_ip>, and B<-verify_email>
 have no effect on the certificate verification enabled via this option.
 
-=item B<-tls_host> I<name>
+=item B<-tls_host> I<address>
```

Comment:
This is a mistake IMHO.  It'll rarely be an IP address, more frequently one might put an address in the URI to deal with NAT or similar, but specify the name in `-tls_host`.  Certificates with IP addresses are quite rare. So this should say `host`, and explain that `host` is typically a DNS name, and sometimes (rarely) an IP address.